OpenSim/Common/Test/testComponentInterface.cpp

```diff
@@ -1614,16 +1614,21 @@ void testTableReporter() {
             TimeStepper ts(system, integ);
             ts.initialize(s);
             ts.stepTo(1.0);
-            std::cout << "TableReporter table after simulating:\n";
             const auto& table = reporter->getTable();
-            SimTK_TEST_MUST_THROW_EXC(table.toString(), EmptyTable);
+            std::cout << "TableReporter table after simulating:\n"
+                      << table.toString() << std::endl;
+            SimTK_TEST_MUST_THROW_EXC(table.getDependentColumnAtIndex(0),
```

Comment:
The PR is already an improvement, so I'm happy for additional changes to come separately. It's pretty low priority but a bit more informative to the user.